microservices/s3_to_redshift/build_derived_assets.py

```diff
@@ -159,7 +191,8 @@
         THEN SUBSTRING (referrer_urlpath,POSITION ('?' IN referrer_urlpath) +1)
         ELSE ''
         END AS referrer_urlquery
-    FROM {schema_name}.asset_downloads AS assets;
+    FROM {schema_name}.asset_downloads AS assets
+    WHERE asset_url NOT LIKE 'https://www.workbc.ca/Contact-Us' AND asset_url NOT LIKE 'https://www.workbc.ca/';
```

Comment:
pycodestyle E501 line too long

Reply:
try instead:
```
    WHERE asset_url NOT LIKE 'https://www.workbc.ca/Contact-Us'
      AND asset_url NOT LIKE 'https://www.workbc.ca/';
```

Reply:
Ive changed the logic, and corrected the line length.